sdk/identity/azure-identity/azure/identity/_credentials/default.py

```diff
@@ -82,7 +82,7 @@ class DefaultAzureCredential(ChainedTokenCredential):
         :class:`~azure.identity.VisualStudioCodeCredential`. Defaults to the "Azure: Tenant" setting in VS Code's user
         settings or, when that setting has no value, the "organizations" tenant, which supports only Azure Active
         Directory work or school accounts.
-    :keyword int developer_credential_timeout: The timeout in seconds to use for developer credentials that run
+    :keyword int process_timeout: The timeout in seconds to use for developer credentials that run
```

Comment:
Can we add a note that it's the timeout for the subprocesses in those credentials?

Reply:
Sorry. Not quite clear.

Currently it is "The timeout in seconds to use for developer credentials that run subprocesses (e.g. AzureCliCredential, AzurePowerShellCredential). Defaults to **10** seconds."

What's your suggestion?